### PR TITLE
#2183 switch to licence expression + repo info

### LINKED ISSUE #2183: Switch to LicenceExpression
### Summary
I wish for the nuget packages to have the licence expression property set as opposed to the licence url property.

### Details

The licence expression property should be set to the correct licence type (Apache-2.0) as this will enable analysis of licences in use to occur in external tools & the license type will be shown in Nuget etc.

### PR DESCRIPTION
**Pull request by thompson-tomo**
This sets additional package info and switch to licence expression rather than url. Also lint csproj to be consistantly tabbed.

closes #2183

### CODE REVIEW

File: src/Confluent.Kafka/Confluent.Kafka.csproj [hunk 1]
```diff
@@ -1,38 +1,40 @@
...
-    <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <SignAssembly>true</SignAssembly>
-    <AssemblyOriginatorKeyFile>Confluent.Kafka.snk</AssemblyOriginatorKeyFile>
+	<ProjectTypeGuids>{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
+	<Authors>Confluent Inc.;Andreas Heider</Authors>
+	<Description>Confluent's .NET Client for Apache Kafka</Description>
+	<Copyright>Copyright 2016-2020 Confluent Inc., Andreas Heider</Copyright>
+	<PackageProjectUrl>https://github.com/confluentinc/confluent-kafka-dotnet/</PackageProjectUrl>
+	<PackageLicenceExpression>Apache-2.0</PackageLicenceExpression>
+	<RepositoryUrl>https://github.com/confluentinc/confluent-kafka-dotnet/</RepositoryUrl>
```
Comment (anchitj) [inline src/Confluent.Kafka/Confluent.Kafka.csproj:9]: Can we update it to `https://github.com/confluentinc/confluent-kafka-dotnet.git` everywhere

File: src/Confluent.Kafka/Confluent.Kafka.csproj [hunk 2]
```diff
@@ -1,38 +1,40 @@
...
+	<RepositoryType>Git</RepositoryType>
```
Comment (anchitj) [inline src/Confluent.Kafka/Confluent.Kafka.csproj:10]: Let's use lowercase `git` everywhere

File: src/Confluent.Kafka/Confluent.Kafka.csproj [hunk 3]
```diff
@@ -1,38 +1,40 @@
...
+	<PackageIconUrl>https://raw.githubusercontent.com/confluentinc/confluent-kafka-dotnet/master/confluent-logo.png</PackageIconUrl>
```
Comment (anchitj) [inline src/Confluent.Kafka/Confluent.Kafka.csproj:11]: Can we please also add `PackageIcon` while keeping this `PackageIconUrl` as well as it seems `PackageIconUrl` is deprecated.

File: src/Confluent.SchemaRegistry/Confluent.SchemaRegistry.csproj [hunk 4]
```diff
@@ -1,33 +1,34 @@
...
-    <SignAssembly>true</SignAssembly>
-    <AssemblyOriginatorKeyFile>Confluent.SchemaRegistry.snk</AssemblyOriginatorKeyFile>
+	<ProjectTypeGuids>{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
+	<Authors>Confluent Inc.</Authors>
+	<Description>A .NET Client for Confluent Schema Registry</Description>
+	<Copyright>Copyright 2017-2022 Confluent Inc.</Copyright>
+	<PackageProjectUrl>https://github.com/confluentinc/confluent-kafka-dotnet/</PackageProjectUrl>
+	<PackageLicenceExpression>Apache-2.0</PackageLicenceExpression>
+	<RepositoryUrl>https://github.com/confluentinc/confluent-kafka-dotnet/</RepositoryUrl>
+	<RepositoryType>Git</RepositoryType><PackageIconUrl>https://raw.githubusercontent.com/confluentinc/confluent-kafka-dotnet/master/confluent-logo.png</PackageIconUrl>
```
Comment (anchitj) [inline src/Confluent.SchemaRegistry/Confluent.SchemaRegistry.csproj:11]: Line break

**Review by anchitj — CHANGES_REQUESTED** [review]
Thanks for the contribution. Left some minor nitpicks.

**Comment by thompson-tomo** [comment]
Will make the changes tomorrow/tonight.

**Review by anchitj — APPROVED** [review]
Thanks for the changes. I've pushed a commit to fix some typos and tabs with spaces.

**Review by emasab — APPROVED** [review]

**Comment by anchitj** [comment]
Tested in https://github.com/confluentinc/confluent-kafka-dotnet/pull/2250

**Review by emasab — APPROVED** [review]

File: src/Confluent.Kafka/Confluent.Kafka.csproj [hunk 5]
```diff
@@ -5,7 +5,10 @@
     <Description>Confluent's .NET Client for Apache Kafka</Description>
     <Copyright>Copyright 2016-2020 Confluent Inc., Andreas Heider</Copyright>
     <PackageProjectUrl>https://github.com/confluentinc/confluent-kafka-dotnet/</PackageProjectUrl>
-    <PackageLicenseUrl>https://github.com/confluentinc/confluent-kafka-dotnet/blob/master/LICENSE</PackageLicenseUrl>
+    <PackageLicenceExpression>Apache-2.0</PackageLicenceExpression>
```
Comment (rogerbriggen-securiton) [inline src/Confluent.Kafka/Confluent.Kafka.csproj:8]: must be PackageLicenseExpression, see https://github.com/NuGet/Home/wiki/Packaging-License-within-the-nupkg#project-properties

**Pull request merged**